src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -34,20 +32,18 @@ public ThemeServiceTests()
         }
 
         [Fact]
-        public async Task GetPublicationTree_Empty()
+        public async Task GetPublicationTree_NoThemes()
```

Comment:
I think the `CacheServiceTestFixture` and `BlobCacheService` setup at the top of this class can all be removed now